1. Basics/task_7/part_3/task_3_on_apache_handler.php

```diff
@@ -0,0 +1,172 @@
+<?php
+/**
+ * Array contains data for response forming according its status code
+ */
+define('RESPONSE_DATA', [
+    '200' => ['statusMessage' => 'OK'],
+    '400' => ['statusMessage' => 'Bad Request'],
+    '404' => ['statusMessage' => 'Not Found'],
+]);
+
+processHttpRequest();
+
+// REQUEST PROCESS FUNCTION ///////////////////////////////////////////////////////////////////////
+/**
+ * Processes http request, partly forms the response. 
+ * Optionally outputs the results as HTML page.
+ * 
+ * @return void
+ */
+function processHttpRequest() : void {
+	$nums = testInputValue($_GET['nums'] ?? false);
+	$invalidRequest = checkRequestCorrectness($nums);
+	$isHTML = preg_match('#html#', $_SERVER['HTTP_ACCEPT'] ?? '');
```

Comment:
same as below... this check in needed later, but not here

Reply:
Done.